src/Application/Clans/Models/UserClanViewModel.cs

```diff
@@ -0,0 +1,12 @@
+using Crpg.Application.Common.Mappings;
+
+namespace Crpg.Application.Clans.Models;
+using Crpg.Application.Common.Mappings;
+using Crpg.Domain.Entities.Clans;
+
+
+public record UserClanViewModel : IMapFrom<ClanMember>
```

Comment:
Can you reorder the statements
1. using
2. namespace
3. record

Reply:
done, sorry, 1st time not having VS hold my hand when creating files, my bad :D